VaticanBAV/et/BAVet125.xml

```diff
@@ -0,0 +1,437 @@
+                                    <locus target="#83v"/>
+                                    <title type="complete" ref="LIT1295Dersan#SalamTeqemt"/>
+                                </msItem>
+                            </msItem>
+                            <msItem xml:id="ms_i1.13">
+                                <locus from="83v" to="86v"/>
+                                <title type="complete" ref="LIT1295Dersan#Introduction"/>
+                              <incipit xml:lang="gez">
+                                  በስመ፡ አብ፡ <gap reason="ellipsis" extent="unknown" resp="PRS4805Grebaut PRS9530Tisseran"/> ስምዑ፡ ሕዝበ፡ ክርስቲያን፡ ወነጽሩ፡ አኃውየ፡ ፍቁራንየ፡ 
+                                  ዕበዮ፡ ወኃይሎ፡ ወመንክሮ፡ ዘገብረ፡ ቅዱስ፡ ሚካኤል፡ ለእለ፡ ይስዕልዎ። በፃሕቅ፡ ወበትውክልተ፡ ልብ፡ <sic>ወበንጽሕ፡</sic> ሥጋ፡ መፍትው፡ ለነ፡ ከመ፡ 
+                                  ንዘከር፡ ወንግበር፡ ተዝካሮ፡ ኵሎ፡ ጊዜ። ከመ፡ ይስአል፡ ወይጸሊ፡ በእንቲአነ፡ ከመ፡ ንርከብ፡ ሞገሰ፡ በቅድመ፡ አዕይንቲሁ፡ ለእግዚአብሔር። አኃዜ፡ ኵሉ፡
+                                  ዓለም፡
+                              </incipit>
+                                <explicit xml:lang="gez">
+                                    እስመ፡ መንበረ፡ ስብሐቲሁ፡ ለንጉሠ፡ ሰማያት፡ ወምድር፡ ይእቲ፡ <surplus>ወ</surplus>በማኅ<supplied reason="omitted">በ</supplied>ረ፡ መላእክት፡ 
+                                    ፍሡሐን፡ ወበሰማዕት፡ ፀዋርያነ፡ አክሊል፡ <gap reason="ellipsis" extent="unknown" resp="PRS4805Grebaut PRS9530Tisseran"/> ወኵሎሙ፡ እሉ፡ ዓሥመርዎ፡
+                                    ለእግዚአብሔር፡ በምግባሮሙ፡ ንጹሕ፡ እምዘርዑ፡ ለአዳም፡ ይእዜኒ፡ ወዘልፈኒ፡ ወለዓለመ፡ ዓለም፡ አሜን፡ ወአሜን።
+                                </explicit>
+                            </msItem>
+                        </msItem>                        
+                    </msContents>
+                    
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">88</measure>
+                                    <measure unit="leaf" type="blank">8</measure>
+                                    <locus from="1r" to="2v"/>
+                                    <locus from="87r" to="88v"/>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>207</height>
+                                        <width>137</width>
+                                    </dimensions>
+                                </extent>
+                                <foliation/>
+                                <collation>
+                                    <signatures>
+                                        <note>
+                                            Quires marks are written with Ethiopic numerals in the upper corner of the recto of the first folio of quires from <ref target="#q3"/> to <ref target="#q7"/>.
```

Comment:
According to the guidelines, content can be written directly inside signatures, the element notes should not be necessary (I don't know whether this causes problems in the visualisition)